librz/core/analysis_tp.c

```diff
@@ -181,16 +181,16 @@ static ut64 get_addr(Sdb *trace, const char *regname, int idx) {
 	return rz_num_math(NULL, sdb_const_get(trace, query, 0));
 }
 
-static _RzAnalysisCond cond_invert(RzAnalysis *analysis, _RzAnalysisCond cond) {
+static _RzTypeCond cond_invert(RzAnalysis *analysis, _RzTypeCond cond) {
```

Comment:
I recommend to make this API function and move to the `librz/type` as well. In `helpers.c` or something similar.